Src/Private/Get-AbrFgtSystem.ps1

```diff
@@ -250,43 +250,204 @@ function Get-AbrFgtSystem {
+                                            "Role"                = $interface.role
+                                            #"MTU"                 = $interface.mtu   # Will be enabled next release when the TableWrite function is added
+                                            "Mode"                = $interface.mode
+                                            "IP Address"          = $interface.ip
+                                            #"Secondary IP"        = $interface.'secondaryip'   # Will be enabled next release when the TableWrite function is added
+                                            "Allow Access"        = $interface.allowaccess
+                                            "Status"              = $interface.status
+                                        }
+                                    }
+                                    # vap-switch falls under default
+                                    Default {
+                                        $OutObj += [pscustomobject]@{
+                                            "Name"                = $interface.name
+                                            "VDOM"                = $interface.vdom
+                                            "Role"                = $interface.role
+                                            "MTU"                 = $interface.mtu
+                                            "VLAN ID"             = $interface.vlanid
+                                            "Mode"                = $interface.mode
+                                            "IP Address"          = $interface.ip
+                                            #"Secondary IP"        = $interface.'secondaryip'   # Will be enabled next release when the TableWrite function is added
+                                            "Allow Access"        = $interface.allowaccess
+                                            "Status"              = $interface.status
+                                        }
+                                    }
+                                }
+                            }
 
-                    $OutObj | Table @TableParams
+                            # VLAN interfaces
+                            if ($interfaceType -eq "vlan") {
+                                $vlanUpCount = ($OutObj | Where-Object { $_.Status -eq 'up' }).Count
+                                $vlanDownCount = ($OutObj | Where-Object { $_.Status -ne 'up' }).Count
+                                Paragraph "Total number of unique VLANs found: $($vlanUpCount + $vlanDownCount), of which $vlanUpCount are up and $vlanDownCount are down."
+                                if ($vlanUpCount -gt 0) {
+                                    $vlanUpIDs = ($OutObj | Where-Object { $_.Status -eq 'up' } | Select-Object -ExpandProperty 'VLAN ID' -Unique)
+                                    Paragraph "- Up VLANs are: $($vlanUpIDs -join ', ')."
```

Comment:
sort by id ? and add also name? (for down interface ?)

Reply:
Less value as the names are already displayed in the table below.
```suggestion
                                    Paragraph "- Up VLANs are: $($vlanUpIDs -join ', ')."
```